packages/shared/src/request/comment.ts

```diff
@@ -0,0 +1,28 @@
+import { useMutation } from 'react-query';
+import { DELETE_COMMENT_MUTATION } from '../graphql/comments';
+import { EmptyResponse } from '../graphql/emptyResponse';
+import { Post } from '../graphql/posts';
+import { usePostComment } from '../hooks/usePostComment';
+import { useRequestProtocol } from '../hooks/useRequestProtocol';
+import { apiUrl } from '../lib/config';
+
+export const deleteComment = ({
```

Comment:
I didnt want to make the main components bloated, so I created this folder/files to keep the request helpers